### DISCUSSION TITLE
LLM Model

### DISCUSSION THREAD

**Discussion by almas9945** [community/community#74388] [Programming Help]
### Hi everyone,

Hi, Can I use BERT, RoBERTa, or XLNet models for my research using the free resources provided by Google Colab?

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @almas9945 , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
